Makefile

```diff
@@ -36,6 +36,9 @@ stage:
 config:
 	scripts/set-config.sh "${config_path}"
 
+secgrp:
```

Comment:
This doesn't need to be a separate call. It should be part of config.
It should naturally know whether it needs to add the secgroups or not depending on user configuration.

Reply:
Done.  Reintegrated with config target.